contributing/setup.md

```diff
@@ -43,12 +43,6 @@ And you'll stay up to date!
 
 ## Protoc
 
```

Comment:
proto-compilation looked to be fixed and merged. This comment may no longer be relevant?